README.md

```diff
@@ -140,6 +140,22 @@ func application(_ application: UIApplication, didFailToRegisterForRemoteNotific
 }
 ```
 
+### Location pushes
+
+Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
+
+To use this new functionality you should obtain Apple entitlments, create location push service extension and make some tweaks in your push notifications code within your app:
+
+* Add Apple location pushes entitlments into your project ("AblyLocationPush" folder).
+
+* In addition to saving your regular device token by calling `ARTPush.didRegisterForRemoteNotifications(withDeviceToken:realtime:)` you should request another device token specifically for the location pushes purpose. You do so by calling `CLLocationManager.startMonitoringLocationPushes(completion:)` within a `ARTPushRegistererDelegate.didActivateAblyPush(:)` delegate callback or whenever is appropriate in your app.
+
+* Once you receive the location push token, save it by calling a new `ARTPush.didRegisterForLocationNotifications(withDeviceToken:realtime:)` method (note the "Location" word in the name of this method).
+
+* To see received location push notifications in the UI you need to update both "AblyPushExample.entitlements" file and the file you received from Apple and add appropriate value under the `com.apple.security.application-groups` key ("group.{your_bundle_identifier}"). It is because application groups is the only way app and its location push extension can exchange information.
```

Comment:
Documentation about how to test location push notifications in the example app should go in the example app’s README, not that of the library.

Reply:
But also these instructions are very vague — what "file you received from Apple"? Why is `group.{your_bundle_identifier}` the appropriate value to use? (It isn't necessarily — the app group identifier is the user’s choice.)

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/794e634e187b3df8f0a56ffac1243e41e592211e